server/@types/ciagInductionApi/index.d.ts

```diff
@@ -26,17 +26,26 @@ export interface paths {
      */
     delete: operations['deleteCIAGProfileForOffenderId']
   }
+  '/ciag/induction/list': {
+    /**
+     * Fetch the CIAG profile for the given offender ids
+     * @description Currently requires role <b>ROLE_VIEW_PRISONER_DATA</b>
+     */
+    post: operations['getAllCIAGProfileForGivenOffenderIds']
+  }
 }
 
 export type webhooks = Record<string, never>
 
 export interface components {
   schemas: {
+    /** @description This is the qualification list of the inmate. */
     AchievedQualification: {
-      subject?: string
-      grade?: string
-      /** @enum {string} */
-      level?:
+      /**
```

Comment:
Lots of noise where the CIAG team have been adding documentation to their API swagger spec